rules/multi_arch_wrapper.bzl

```diff
@@ -0,0 +1,65 @@
+"""
+Wrap a macOS tool in a multi-arch transition. This is useful for building fat
+binaries to share caches between Apple Silicon and Intel machines. This
+behavior requires Xcode 12
+"""
+
+# NOTE: This lives in apple_support instead of rules_apple so that rules_swift
+# can depend on it
+
+def _force_multi_arch_transition_impl(settings, attr):
+    return {"//command_line_option:macos_cpus": "arm64,x86_64"}
```

Comment:
Tried this but it doesn't work for host tools for me. Seems like host compilation doesn't read the `--macos_cpus` flag..

Reply:
With this transition it doesn't respect it at all and just hard codes this. Are you saying that you're still not getting fat binaries if it's host?

Reply:
I'm not getting fat binaries if it's host (only tried the transition, not the rule)

Reply:
I'll have to test again, I thought this worked for host tools but maybe it did or something regressed

Reply:
It works if I use the rule directly.